rust/ql/lib/codeql/rust/controlflow/internal/ControlFlowGraphImpl.qll

```diff
@@ -507,6 +538,10 @@ class TypeRefTree extends LeafTree instanceof TypeRef { }
 
 class UnderscoreExprTree extends LeafTree instanceof UnderscoreExpr { }
 
+class UseTree_ extends StandardPreOrderTree, Use {
```

Comment:
A `use` node is an item and is not something that gets evaluated as far as I know.

Reply:
As discussed offline, let's make all `Item`s leaf nodes for now.